tests/datastore/test_targets.py

```diff
@@ -111,3 +112,25 @@ def test_write_with_too_many_partitions():
         match="Maximum number of partitions exceeded. To resolve this.*",
     ):
         parquet_target.write_dataframe(df)
+
+
+def test_revert_list_filters_to_tuple():
+    #  Target-specific to test_revert_list_filters_to_tuple test, which checks all the cases.
+    def json_change(filters):
+        json_data = json.dumps(filters)
+        return json.loads(json_data)
+
+    additional_filters = [[("x", "=", 3), ("x", "=", 4), ("x", "=", 5)]]
+    parquet_target = ParquetTarget("parquet_target", path="path/to/file")
+    after_json_change_filters = json_change(additional_filters)
```

Comment:
Just inline it... "json change" doesn't make it any more readable.
```suggestion
    back_from_json_serialization = json.loads(json.dumps(additional_filters))
```

Reply:
tnx!